harvard-coventry-university.csl

```diff
@@ -284,7 +280,7 @@
     <layout prefix="(" suffix=")" delimiter=", ">
       <choose>
         <if locator="page">
-          <group delimiter=":&#160;">
+          <group delimiter=": ">
```

Comment:
@adam3smith, before you merge this, we might want to make certain these non-breaking spaces are still there, and escape them.